app/assets/stylesheets/components/_select-with-search.scss

```diff
@@ -0,0 +1,416 @@
+.app-c-select-with-search {
+  //
+  // A theme for Choices.js styled to look like a GOV.UK Design System component
+  //
+  $background: #ffffff;
+  $focus-color: $govuk-focus-colour;
+  $focus-outline: $govuk-focus-width solid $govuk-focus-colour;
+
+  .choices * {
+    // Something inside .choices needs this – I'm not sure what yet
+    box-sizing: border-box;
+  }
+
+  .choices {
+    @include govuk-font($size: 19);
+    position: relative;
+    z-index: 2;
+    margin-bottom: 24px;
+  }
+
+  .choices:focus {
+    outline: none;
+  }
+
+  .choices:last-child {
+    margin-bottom: 0;
+  }
+
+  // Disabled state
+  .choices.is-disabled {
+    .choices__inner,
+    .choices__input {
+      background-color: #eaeaea;
+      cursor: not-allowed;
+      -webkit-user-select: none;
+      user-select: none;
+    }
+
+    .choices__item {
+      cursor: not-allowed;
+    }
+  }
+
+  // This is an intentionally high specificity selector to avoid using !important when targetting
+  // the <select hidden> element that would otherwise end up picking up `display: block` from the
+  // `.choices[data-type*="select-one"] .choices__input` rule below.
+  .choices .choices__inner select[hidden] {
+    display: none;
+  }
+
+  // Type: Single Select
+  .choices[data-type*="select-one"] {
+    cursor: pointer;
+
+    .choices__input {
+      display: block;
+      width: 100%;
+      padding: 10px;
+      border-bottom: 1px solid #dddddd;
+      background-color: $background;
+      margin: 0;
+    }
+
+    .choices__button {
+      background-image: url("data:image/svg+xml,%3Csvg width='21' height='21' viewBox='0 0 21 21' xmlns='http://www.w3.org/2000/svg'%3E%3Cg fill='%23000' fill-rule='evenodd'%3E%3Cpath d='M2.592.044l18.364 18.364-2.548 2.548L.044 2.592z'/%3E%3Cpath d='M0 18.364L18.364 0l2.548 2.548L2.548 20.912z'/%3E%3C/g%3E%3C/svg%3E");
```

Comment:
could we move this to a file or not possible?

Reply:
To be honest, I don't think `.choices__button` ever gets created. It's used when Choices.js is in 'multiple select' mode – but the Whitehall component only ever runs it in 'single select' mode.

However I don't really want to go through and strip support for 'multiple select' from the theme. I'd rather leave this intact as is, where the SVG lives directly where it's used, and the theme supports all the potential use cases of the JS library (in case our needs ever change in the future).